onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1348,6 +1348,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "Number of attention heads. Default value is 0. Must use with rotary_embedding_dim",
               AttributeProto::INT,
               OPTIONAL_VALUE)
+        .Attr("is_batch_inputs_packed",
```

Comment:
Should the attribute name be something like `is_packed_batching` instead?

Reply:
Sounds good